packages/cli/src/cmds/record/test/isAgentAvailable.ts

```diff
@@ -9,9 +9,9 @@ export default async function isAgentAvailable(): Promise<boolean> {
   try {
     await new RemoteRecording(ro).status();
     UI.success(`AppMap agent is available at ${location}`);
-     return true;
+    return true;
   } catch (e: any) {
-    UI.error(`AppMap agent is not avaliable at ${location}: ${e.toString()}`);
+    UI.error(`AppMap agent is not available at ${location}: ${e.toString()}`);
     return false;
```

Comment:
I considered this. I didn't do it, because a 404 might also just indicate that the user fat-fingered the server's URL. So, it seems like the flow needs to change a little more: if we see a 404, give the user the choice to either update the URL and try again, or to open a ticket. If we see some other non-200 status, we can just offer to open a ticket.

WDYT?

Reply:
That sounds like a good solution to me!

Reply:
New 404 flow: https://asciinema.org/a/FzZNBRE3WVJBfEs5h0cuqraD8